### PR TITLE
PMM-2433, PMM-2434: Add new options for slow log rotation.

### PR DESCRIPTION
**Pull request by arvenil**
Add new options:
* --disable-slow-log-rotation
* --retain-slow-logs=5

### CODE REVIEW

File: pmm/mysql_queries.go [hunk 1]
```diff
@@ -22,91 +22,104 @@ import (
...
 
+// MySQLQueriesFlags MySQL Queries specific flags.
+type MySQLQueriesFlags struct {
+	QuerySource string
+	// slowlog specific options.
+	RetainSlowLogs          int
+	DisableSlowLogsRotation bool
+}
+
+// MySQLQueriesAddResult is result returned by AddMySQLQueries.
```
Comment (golangci) [inline pmm/mysql_queries.go]: comment on exported type MySQLQueriesResult should be of the form "MySQLQueriesResult ..." (with optional leading article)

File: pmm-admin_test.go [hunk 2]
```diff
@@ -1299,6 +1305,352 @@ func testAddLinuxMetricsWithAdditionalArgsFail(t *testing.T, data pmmAdminData)
...
+					panic(fmt.Sprintf("error unmarshaling body: %s", err))
+				}
+				err = json.Unmarshal(cmd.Data, &config)
+				if err != nil {
+					panic(fmt.Sprintf("error unmarshaling body: %s", err))
+				}
+
+				w.WriteHeader(http.StatusOK)
+			default:
+				w.WriteHeader(600)
```
Comment (percona-csalguero) [inline pmm-admin_test.go]: 600?

File: pmm/mysql.go [hunk 3]
```diff
@@ -267,21 +253,10 @@ func testConnection(dsn string) error {
...
-	return map[string]string{
-		"hostname":    hostname,
-		"port":        port,
-		"distro":      distro,
-		"version":     version,
-		"table_count": tableCount,
-	}
+func getMysqlInfo(db *sql.DB) *MySQLInfo {
+	mi := &MySQLInfo{}
+	db.QueryRow("SELECT @@hostname, @@port, @@version_comment, @@version").Scan(&mi.Hostname, &mi.Port, &mi.Distro, &mi.Version)
```
Comment (golangci) [inline pmm/mysql.go]: Error return value of `(*database/sql.Row).Scan` is not checked

File: pmm-admin_test.go (hunk 2 above)
Comment (arvenil) [reply pmm-admin_test.go]: I just did it to indicate this is unsupported call, in contrast to returning 500 which could be a real response. Maybe `panic` would be better here. What do you think?

File: pmm/mysql.go (hunk 3 above)
Comment (golangci) [inline pmm/mysql.go]: Error return value of `(*database/sql.Row).Scan` is not checked

**Pull request merged**